Sources/Concrete/ViewModel/RefinementMenuViewModel.swift

```diff
@@ -111,8 +111,22 @@ public class RefinementMenuViewModel: RefinementMenuViewModelDelegate, Searchabl
     /// it will tggle the facet refinement, deselect the row and then execute a search
     public func didSelectRow(at indexPath: IndexPath) {
         
-        searcher.params.setFacet(withName: attribute, disjunctive: isDisjunctive)
-        searcher.params.toggleFacetRefinement(name: attribute, value: facetResults[indexPath.item].value)
+        searcher.params.setFacet(withName: attribute, disjunctive: true)
```

Comment:
Why true and not `isDisjunctive `

Reply:
@robertmogos  When showing a refinement list, if you select a refinement value, then what happens is that we get a result set that doesn't contain all other possible refinement values. (imagine we have 3 types: `electronics` `gifts` `books`. if we put nonDisjunctive, then when selecting `books`, the user won't be able to see the other 2 options. if we put it to `true`, we receive the other 2 possible types.

So in order to solve this, I moved to an **always disjunctive mode** (`true` instead of `isDijunctive`, and when the user selects another type, I make sure that at most 1 item is selected.